app/src/main/assets/special_thanks.csv

```diff
@@ -1,4 +1,6 @@
-221 Pixels;Logo design;https://avatars2.githubusercontent.com/u/58243143?s=60&v=4
-Anxhelo Lushka;Website design;https://avatars2.githubusercontent.com/u/25004151?s=60&v=4
-ByteHamster;Forum admin;https://avatars2.githubusercontent.com/u/5811634?s=60&v=4
-Keunes;Communications;https://avatars2.githubusercontent.com/u/11229646?s=60&v=4
+ByteHamster;Project lead;https://avatars2.githubusercontent.com/u/5811634?s=60&v=4
+Keunes;Project lead;https://avatars2.githubusercontent.com/u/11229646?s=60&v=4
```

Comment:
How about something like `Project lead (Development)` and `Project lead (Communications)`? Or is that too detailed?

Reply:
I think it won't work nicely, because we do more than that. You also administer the forum, I also do UI design, etc. It'd become a rather long list, so I stuck to this.